scripts/ci/run-jenkins.sh

```diff
@@ -6,8 +6,14 @@ export TEST_HARNESS_VERBOSE=1
 
 if [[ ${label} == 'osx-i386' ]]; then EXTRA_CONF_FLAGS="--with-libgdiplus=/Library/Frameworks/Mono.framework/Versions/Current/lib/libgdiplus.dylib --enable-nls=no --build=i386-apple-darwin11.2.0"; fi
 if [[ ${label} == 'osx-amd64' ]]; then EXTRA_CONF_FLAGS="--with-libgdiplus=/Library/Frameworks/Mono.framework/Versions/Current/lib/libgdiplus.dylib --enable-nls=no"; fi
-if [[ ${label} == 'w32' ]]; then PLATFORM=Win32; EXTRA_CONF_FLAGS="--host=i686-pc-mingw32"; export MONO_EXECUTABLE="`cygpath -u ${WORKSPACE}\\\msvc\\\Win32\\\bin\\\Release_SGen\\\mono-sgen.exe`";fi
-if [[ ${label} == 'w64' ]]; then PLATFORM=x64; EXTRA_CONF_FLAGS="--host=i686-pc-mingw32"; export MONO_EXECUTABLE="`cygpath -u ${WORKSPACE}\\\msvc\\\x64\\\bin\\\Release_SGen\\\mono-sgen.exe`"; fi
+if [[ ${label} == 'w32' ]]; then PLATFORM=Win32; EXTRA_CONF_FLAGS="--host=i686-pc-mingw32"; fi
+if [[ ${label} == 'w64' ]]; then PLATFORM=x64; EXTRA_CONF_FLAGS="--host=i686-pc-mingw32"; fi
+
+if [[ ${label} == w* ]]; then
+	export MONO_EXECUTABLE="`cygpath -u ${WORKSPACE}\\\msvc\\\\${PLATFORM}\\\bin\\\Release_SGen\\\mono-sgen.exe`"
+	# without this dir in PATH the P/Invoke tests in mono/tests/ won't find the MSVC built libtest.dll
+	export PATH="$PATH:`cygpath -u ${WORKSPACE}\\\msvc\\\\${PLATFORM}\\\bin\\\Release`"
```

Comment:
Hmm, indeed. We should get a `--config` option added to the managed test-runner.cs then which passes this along.

(You can try if it works at all by copying the config file as `<appname>.exe.config` next to the exe, then it should get automatically used)